hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3070,13 +3068,10 @@ public TenantUserList listUsersInTenant(String tenantId, String prefix)
       return null;
     }
 
-    if (!multiTenantManager.tenantExists(tenantId)) {
-      // Throw exception to the client, which will handle this gracefully
-      throw new OMException("Tenant '" + tenantId + "' not found",
-          TENANT_NOT_FOUND);
-    }
+    OMMultiTenantManagerImpl.checkTenantExistence(
```

Comment:
IMO we should avoid calling Impl directly. Either:
1. use `static` method in the interface (when applicable), or
2. declare abstract methods in the interface and call that instead.

Reply:
I agree. I think the methods moved from OMTenantRequestHelper should be instance methods declared in the interface. All places using them should be able to get an OMMultitenantManager instance from the OzoneManager. Having static methods in OMMultitenantManagerImpl that take an OMMetadataManager doesn't really make sense because the OMMultitenantManager instance already has one as a field.

Reply:
I have updated the PR with the above suggestions. Thank you.

Reply:
Thanks @aswinshakil I will take a look. In the mean time you could resolve the merge conflicts.

Reply:
Sure I will resolve it.